lib/workload/stateless/stacks/filemanager/filemanager/src/env.rs

```diff
@@ -46,6 +53,29 @@ pub struct Config {
     pub(crate) api_cors_allow_headers: Vec<String>,
 }
 
+/// Default presigned URL expiry time, 5 minutes.
+pub const DEFAULT_PRESIGN_EXPIRY: Duration = Duration::hours(1);
```

Comment:
Is the presign here defaulting to one hour instead of 12? or I just can't read Rust code?

Reply:
Yeah, it defaults to one hour. I'll change it to 12 hours so it's consistent everywhere.

Reply:
The deploy code is overriding that [here](https://github.com/umccr/orcabus/blob/5a77bec8f1bc534eaed2c315ddb740c0e1585eb3/lib/workload/stateless/stacks/filemanager/deploy/constructs/functions/api.ts#L23).

Reply:
> `/// Default presigned URL expiry time, 5 minutes.`

That comment is also not very useful! :laughing:

Reply:
> The CDK code is overriding that [here](https://github.com/umccr/orcabus/blob/5a77bec8f1bc534eaed2c315ddb740c0e1585eb3/lib/workload/stateless/stacks/filemanager/deploy/constructs/functions/api.ts#L23)

All good then, I wasn't aware of the overriding bit here.